plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftClient.java

```diff
@@ -230,6 +232,18 @@ public class RedshiftClient
     private final ConnectorExpressionRewriter<ParameterizedExpression> connectorExpressionRewriter;
     private final Optional<Integer> fetchSize;
 
+    private static PredicatePushdownController getPushdownController(Type type)
```

Comment:
You can move this to `io.trino.plugin.jdbc.PredicatePushdownController` and call it `PUSHDOWN_DISCRETE_VALUES`